docs/sources/send-data/promtail/cloud/ec2/_index.md

```diff
@@ -258,7 +259,7 @@ Note that you can use [relabeling][relabeling] to convert systemd labels to matc
 
 That's it, save the config and you can `reboot` the machine (or simply restart the service `systemctl restart promtail.service`).
 
-Let's head back to Grafana and verify that your Promtail logs are available in Grafana by using the [LogQL]({{< relref "../../../query" >}}) query `{unit="promtail.service"}` in Explore. Finally make sure to checkout [live tailing][live tailing] to see logs appearing as they are ingested in Loki.
+Let's head back to Grafana and verify that your Promtail logs are available in Grafana by using the [LogQL]({{< relref "../../../../query" >}}) query `{unit="promtail.service"}` in Explore. Finally make sure to checkout [live tailing][live tailing] to see logs appearing as they are ingested in Loki.
 
 [promtail]: ../../promtail/README
```

Comment:
Unrelated to the PR changes: I'm not sure these relative links behave correctly in the website, they should use the `relref` shortcode if they are to documentation pages, or fully qualified URLs if they are linking to files in the GitHub repository.

There's a few kinds of that link in this file that might make sense to double check.